examples/demo_project/docker-compose.yml

```diff
@@ -0,0 +1,93 @@
+version: "3"
+
+
+services:
+  # database for master
+  db_pgsql:
+    image: postgres:12
+    environment:
+      POSTGRES_USER: master_service
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: master_service
+    volumes:
+      - pgsql_data:/var/lib/postgresql/data
+
+  # database for replica
+  db_mysql:
+    image: mysql:8.0
+    environment:
+      MYSQL_ROOT_PASSWORD: password
+      MYSQL_USER: replica_service
+      MYSQL_PASSWORD: password
+      MYSQL_DATABASE: replica_service
+    volumes:
+      - mysql_data:/var/lib/mysql
+    command: --default-authentication-plugin=mysql_native_password
+
+  # cache storage for replica
+  redis:
+    image: redis
+
+  # CQRS transport backend
+  rabbitmq:
+    image: rabbitmq:3-management-alpine
```

Comment:
I would ask to add to readme info that RabbitMQ admin can be used to observe queues with CQRS data.

Reply:
added